File: Tyche.iOS/Bet/Sources/Bet/PoolGamblerBetTransformer.swift
import DataBet

extension PoolGamblerBet {
    func toPoolGamblerBetModel() -> PoolGamblerBetModel {
        return PoolGamblerBetModel(
            poolId: self.poolId,
            gamblerId: self.gamblerId,
            matchId: self.matchId,
            homeTeamId: self.homeTeamId,
            homeTeamName: self.homeTeamName,
            awayTeamId: self.awayTeamId,
            awayTeamName: self.awayTeamName,
            matchScore: self.matchScore,
            betScore: self.betScore,
            score: self.score,
            matchDateTime: self.matchDateTime,
            isLocked: self.isLocked
        )
    }
}

File: Tyche.Android/bet/src/main/java/com/felipearpa/tyche/bet/PoolGamblerBetTransformer.kt
package com.felipearpa.tyche.bet

import com.felipearpa.tyche.data.bet.domain.PoolGamblerBet

fun PoolGamblerBet.toPoolGamblerBetModel() =
    PoolGamblerBetModel(
        poolId = this.poolId,
        gamblerId = this.gamblerId,
        matchId = this.matchId,
        homeTeamId = this.homeTeamId,
        homeTeamName = this.homeTeamName,
        awayTeamId = this.awayTeamId,
        awayTeamName = this.awayTeamName,
        matchScore = this.matchScore,
        betScore = this.betScore,
        score = this.score,
        matchDateTime = this.matchDateTime,
        isLocked = this.isLocked
    )
